prometheus-exporters/ucs-exporter/alerts/portlicense.alerts

```diff
@@ -3,7 +3,7 @@ groups:
   rules:
   - alert: UcsServerPortLicenseExpired
     expr: >
-      ucs_server_license{port_name=~".+"} == 0
+      max_over_time(ucs_server_license{port_name=~".+"}[15m]) == 0
```

Comment:
15 will not cut it, scrape interval is at 15m, I'd expect 15m+scrape time to be the max diff. I'd put it on 20.

Reply:
Sure.